src/cloud-api-adaptor/docs/SecureComms.md

```diff
@@ -37,49 +37,106 @@ Deploy Trustee-Operator by following instructions at [trustee Operator Getting S
 
 Make sure to uncomment the secret generation as recommended for both public and private key (`kbs-auth-public-key` and `kbs-client` secrets). 
 
-Copy the kbs-client secret from the `kbs-operator-system` namespace to the `confidential-containers-system` ns. This can be done using:
+```sh
+kubectl get secrets -n trustee-operator-system
+NAME                  TYPE     DATA   AGE
+kbs-auth-public-key   Opaque   1      28h
+kbs-client            Opaque   1      28h
+```
```

Comment:
What's the reason for this command being added?

Reply:
This is a slight improvement to the SecureComms doc which shows the correct result after following the instructions trustee operator and following the recommendation: "Make sure to uncomment the secret generation as recommended for both public and private key (`kbs-auth-public-key` and `kbs-client` secrets). "

We can add it to #2073 if it will make things clearer

Reply:
I prefer to leave this extra documentation detail  as is, although it also appears in #2065 - unless someone finds this very disturbing.